sn_node/src/node/messaging/proposal.rs

```diff
@@ -173,6 +181,56 @@ impl MyNode {
                     }
                 }
             }
+            Ok(Either::Right((serialised_proposal_1, serialised_proposal_2))) => {
+                let sig_share2 = if let Some(sig) = optional_sig_share {
+                    sig
+                } else {
+                    error!("Second sig share is missing! {}, {:?}", sender, msg_id,);
+                    return Ok(cmds); // TODO: return error here
+                };
+
+                let res1 = self
+                    .proposal_aggregator
+                    .try_aggregate(&serialised_proposal_1, sig_share);
+                let res2 = self
+                    .proposal_aggregator
+                    .try_aggregate(&serialised_proposal_2, sig_share2);
+                let res = (res1, res2);
+
+                match res {
+                    (Ok(Some(sig1)), Ok(Some(sig2))) => match proposal {
+                        Proposal::HandoverCompleted(SapCandidate::SectionSplit(sap1, sap2)) => cmds
+                            .push(Cmd::HandleNewSectionsAgreement {
+                                sap1,
+                                sig1,
+                                sap2,
+                                sig2,
+                            }),
+                        _ => error!(
+                            "Inconsistent results when aggregating proposal from {}, {:?}",
+                            sender, msg_id,
+                        ),
+                    },
+                    (Ok(None), Ok(None)) => {
+                        trace!(
+                            "Proposals from {} inserted in aggregator, not enough sig shares yet: {serialised_proposal_1:?} and {serialised_proposal_2:?} {:?}",
+                            sender,
+                            msg_id);
+                    }
+                    (_, Err(error)) | (Err(error), _) => {
+                        error!(
+                            "Failed to add proposal from {}, {:?}: {:?}",
+                            sender, msg_id, error
+                        );
```

Comment:
You can save 3 lines by in-lining the variables
```suggestion
                        error!("Failed to add proposal from {sender}, {msg_id:?}: {error:?}");
```